src/client.ts

```diff
@@ -101,30 +130,26 @@ export interface IPageData {
     title: string;
 }
 
-/** One page entry in a layout tree. */
-export interface ITreeEntry {
-    depth: number;
-    route: string;
+/*
+LAYOUT HIERARCHY NODES
```

Comment:
these gigantic capitalized headers indicate to me that these should be a in a separate named "layoutHierarchy.ts" or something